src/test/java/com/dsrm/dsrmbackend/RoomControllerTests.java

```diff
@@ -22,6 +23,7 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
 
 @Testcontainers
+@DirtiesContext
```

Comment:
Why did you add this?

Reply:
I couldn't run all the tests together without it, because TestContainers kills a container and creates a new one. SpringBootTest is not aware of that and tries to connect to an unavailable port